app/models/message.rb

```diff
@@ -0,0 +1,8 @@
+class Message < ApplicationRecord
```

Comment:
I would be nice to have a validation on this model to check that the user/sender is part of the conversation

Reply:
Yes, it would be ready for next PR